keps/sig-storage/1495-volume-populators/README.md

```diff
@@ -105,21 +91,48 @@ flexibility will allow us to experiment with approaches for data populators.
 - Enable users to create pre-populated volumes in a manner consistent with current practice
 - Enable developers to innovate with new an interesting data sources
 - Avoid changing existing workflows and breaking existing tools with new behavior
+- Ensure that users continue to get accurate error messages and feedback about volume
+  creation progress, whether using existing methods of volume creation or specifying a
+  data source associated with a volume populator.
 
 ### Non-Goals
 
 - This proposal DOES NOT recommend any specific approach for data populators. The specific
   design of how a data populator should work will be handled in a separate KEP.
-- We do not propose any new object types. Specifically, no special "populator"
-  object type. Populators should function based on their own object types.
 
 ## Proposal
 
-The validation for `DataSource` field should be removed entirely, in the fullness of time.
-
-Short term, we should add a new alpha feature gate, which allows arbitrary objects to be
-specified for the `DataSource` field, with the intention to eventually make this behavior
-the standard. 
+Validation for the `DataSource` field should be moved from the core to a validation
+webhook. Because the existing validation only allow snapshots and volumes as data sources,
```

Comment:
snapshots and volumes -> VolumeSnapshot and PVC API objects to be more precise

Reply:
Will fix